### DISCUSSION TITLE
1.0 Initial release

### DISCUSSION THREAD

**Discussion by matmair** [matmair/inventree-meta-plugin#1] [Announcements]
First running/runnable unified action to enable pinned/secure action references in accordance with GitHub's best practices.

<hr /><em>This discussion was created from the release <a href='https://github.com/matmair/inventree-meta-plugin/releases/tag/v1.0'>1.0 Initial release</a>.</em>

**Comment by matmair** [comment]
@wolflu05 this might be intresting for you if you create a complexer plugin. Together with the codebase in https://github.com/matmair/inventree-shopify it shows how the pieces can fit together with larger plugins.

**Reply by wolflu05** [reply]
Interesting. I wanted to go into testing if I have my first draft working.

But I have one question left, have you solved the migration issue? From your commits it seems like your plugin is working?

**Reply by matmair** [reply]
As said before I can not reliably reproduce your issue. There will be an update on the [issue](https://github.com/inventree/InvenTree/issues/3549) if I find something.

**Reply by wolflu05** [reply]
@matmair So just for reference. Your Shopify plugin is doing its migrations while invoking migrate?

**Reply by matmair** [reply]
Yes, how else should it work?

**Reply by wolflu05** [reply]
it can't work currently due to a regression, see https://github.com/inventree/InvenTree/issues/3549#issuecomment-1219349744

**Reply by matmair** [reply]
Works on my infra
